Lib/test/test_code.py

```diff
@@ -723,6 +723,7 @@ def f():
             pass
         PY_CODE_LOCATION_INFO_NO_COLUMNS = 13
         f.__code__ = f.__code__.replace(
+            co_stacksize=1,
```

Comment:
Why was this necessary?

Reply:
The function `f`:
```
def f():
    pass

LOAD_CONST 0 (None) # The stack size is 1
RETURN_VALUE
```

After optimization
```
RETURN_CONST 0 (None) The stack size is 0
```

So need to specify the `co_stacksize` when `replace` call.